### PR TITLE
update nodejs 6.14.0. 8.11.0 npm openwhisk 3.14

### LINKED ISSUE #3508: update nodejs with latest security and npm openwhisk 3.14
https://nodejs.org/en/blog/vulnerability/march-2018-security-releases/

### PR DESCRIPTION
**Pull request by csantanapr**
update nodejs 6.14.0. 8.11.0 npm openwhisk 3.14

## Related issue and scope
Closes #3508 

## My changes affect the following components
- [x] Runtimes
- [x] Documentation

## Types of changes
- [x] Runtime Currency

## Checklist:
- [x] I signed an [Apache CLA](https://github.com/apache/incubator-openwhisk/blob/master/CONTRIBUTING.md).
- [x] I reviewed the [style guides](https://github.com/apache/incubator-openwhisk/wiki/Contributing:-Git-guidelines#code-readiness) and followed the recommendations (Travis CI will check :).
- [x] I updated the documentation where necessary.

### CODE REVIEW

**Comment by csantanapr** [comment]
⏳ Playground4/1548/

**Comment by csantanapr** [comment]
🔵 Playground4/1548/

File: docs/reference.md [hunk 1]
```diff
@@ -207,12 +207,12 @@ For the `whisk.error()` you can return a rejected Promise (i.e. Promise.reject).
 ### JavaScript runtime environments
 
 JavaScript actions can be executed in Node.js version 6 or Node.js version 8.
-Currently actions are executed by default in a Node.js version 6.12.2 environment.  
+Currently actions are executed by default in a Node.js version 6.12.2 environment.
```
Comment (dgrove-oss) [inline docs/reference.md]: Should also version bump to 6.14.0 here?

Comment (csantanapr) [reply docs/reference.md]: Yes thank you

Comment (csantanapr) [reply docs/reference.md]: @dgrove-oss I made it generic just 6 since below it talks about the specific version

**Comment by csantanapr** [comment]
@dgrove-oss Travis is now ✅

**Pull request merged**